src/main/java/com/minecolonies/coremod/commands/NavigationMenu.java

```diff
@@ -0,0 +1,29 @@
+package com.minecolonies.coremod.commands;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.jetbrains.annotations.NotNull;
+
+public class NavigationMenu extends AbstractMenu
+{
+    @NotNull private final List<IMenuType> childrenMenuList;
+
+    public NavigationMenu(@NotNull final String menuItemName, @NotNull final IMenuType ... childMenuTypes)
+    {
+        super(menuItemName);
+        this.childrenMenuList = Arrays.asList(childMenuTypes);
+    }
+
+    @Override
+    public NavigationMenuType getMenuType()
+    {
+        return (NavigationMenuType) menuType;
+    }
+
+
+    public List<IMenuType> getChildrenMenuList()
+    {
+        return childrenMenuList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "childrenMenuList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)